feat: add first draft

eddyu0 opened the pull request.

eddyu0 committed:
feat: initial commit

Commit:
feat: add first draft

eddyu0 committed:
feat: use jekyll-theme-slate

tim-hm committed:
chore: add codeowners and gitignore

tim-hm commented:
I've setup docs.glimpsezero.io as a CNAME to avoid the ugly glimpse-protocol.github.io url

tim-hm committed:
feat: add drawio diagram file

tim-hm committed:
chore: add markdown lint config

tim-hm committed:
feat: restructure docs as markdown

This drops github pages setup in preference for a simpler markdown approach.

Added new diagrams for the introduction and refined getting started diagrams.

Added more structure to docs overall.
